platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/MapboxApplication.java

```diff
@@ -4,7 +4,7 @@
 import android.os.StrictMode;
 import android.text.TextUtils;
 import com.mapbox.mapboxsdk.Mapbox;
-import com.mapbox.mapboxsdk.maps.Telemetry;
+import com.mapbox.mapboxsdk.telemetry.Telemetry;
```

Comment:
As mentioned in https://github.com/mapbox/mapbox-gl-native/pull/12468#issuecomment-407669765, this and all other `Telemtry.java` changes are semver major.